rust/src/main.rs

```diff
@@ -63,13 +63,13 @@ fn main() -> anyhow::Result<()> {
         return Ok(());
     }
 
-    let config: Config = if let Some(path) = args.configuration_file.as_deref() {
-        Figment::new().merge(Toml::file(path))
-    } else {
-        Figment::new()
-    }
-    .extract()
-    .unwrap_or_default();
+    let config: Config = Figment::new()
```

Comment:
Why this change?  This will cause an error if no config file is specified and $PWD/pjdfstest.toml does not exist.

Reply:
Seems to be a merge error.